deepchem/models/torch_models/tests/highway_test.py

```diff
@@ -0,0 +1,57 @@
+import torch
+import torch.nn as nn
+import torch.nn.init as init
+import torch.nn.functional as F
+def test_highway():
+  """Test invoking Highway."""
+  width = 5
+  batch_size = 2
+  input = torch.rand(batch_size, width, dtype=torch.float32)
+  layer = Highway(layer_shape=[width, 2])
+  result = layer(input)
+  assert result.shape == (batch_size, width)
+  assert len(layer.parameters()) == 4
+
+  # Creating a second layer should produce different results, since it has
+  # different random weights.
```

Comment:
split this into a separate test and use random seed to avoid randomness while testing

Reply:
Done